### PR TITLE
Add Book Model

### PR DESCRIPTION
**Pull request by Casanova569**
**Summary**
The card is considered approved if after create the model for books, the application lifts and responds upon entering the url in  "localhost: 8081".

**Trello Card**
[Crear un Modelo de Books](https://trello.com/c/LsJDu0XT/14-crear-modelo-de-book)

### CODE REVIEW

File: src/main/java/wolox/training/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import org.springframework.data.annotation.Id;
+
+public class Book {
```
Comment (rodrigofranc) [inline src/main/java/wolox/training/models/Book.java:8]: All attributes should be lower case

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+	@Id
```
Comment (rodrigofranc) [inline src/main/java/wolox/training/models/Book.java:10]: Getter method for this is missing

**Review by rodrigofranc — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/models/Book.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private long Id;
+
+	@Column()
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/models/Book.java]: Just as a comment, the parenthesis aren't necessary here :)

File: src/main/java/wolox/training/models/Book.java [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	public void setPages(Integer pages) {
+		this.Pages = pages;
+	}
+
+	public String getIsbn() {
+		return this.Isbn;
+	}
+	public void setIsbn(String isbn) {
+		this.Isbn = isbn;
+	}
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/models/Book.java:121]: Maybe you could add a getter (however, not a setter) for the book id. 
Another thing you could consider is using Lombok for the getters and setters, to help with writing the class :) just a suggestion for the future. See [this link](http://www.javabyexamples.com/lombok-data-and-value-for-fast-pojos/) as reference

File: src/main/java/wolox/training/models/Book.java [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+	@Column(nullable = false)
+	private Integer Pages;
+
+	@Column(nullable = false)
+	private String Isbn;
+
+	public Book() { }
+
+	public Book(String genre, String author, String image, String title, String subtitle,
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/models/Book.java]: I think you could add `final` to each parameter

**Review by Jimenarosello — CHANGES_REQUESTED** [review]

**Review by rodrigofranc — APPROVED** [review]

**Review by Jimenarosello — APPROVED** [review]
😃

**Pull request merged**